src/js/components/state/topFive/TopFive.jsx

```diff
@@ -31,15 +42,249 @@ const TopFive = (props) => {
+                        },
+                        display: {
+                            entity: "Current congressional district",
+                            standalone: `${linkData._name}, ${location.state}`,
+                            title: linkData._name
+                        }
+                    }
+                }
+            };
+        }
+        else if (params.category === 'cfda') {
+            categoryFilter = {
+                selectedCFDA:
+                    {
+                        [linkData._code]: {
+                            program_number: linkData._code,
+                            program_title: linkData._name,
+                            identifier: linkData._code
+                        }
+                    }
+            };
+        }
+        else if (params.category === 'naics') {
+            categoryFilter = {
+                naicsCodes: {
+                    require: [linkData._code],
+                    exclude: [],
+                    counts: [
+                        {
+                            label: linkData._name,
+                            value: linkData._code,
+                            count: 1
+                        }
+                    ]
+                }
+            };
+        }
+        else if (params.category === "awards") {
+            categoryFilter = {
+                selectedAwardIDs: {
+                    [linkData._name]: linkData._name
+                }
+            };
+        }
+
+        let awardTypeFilter;
+
+        if (params.filters.award_type_codes?.length > 0) {
+            awardTypeFilter = {
+                awardType: params.filters.award_type_codes
+            };
+        }
+
+        const timePeriodFilter = {
+            timePeriodStart: params.filters.time_period[0].start_date,
+            timePeriodEnd: params.filters.time_period[0].end_date,
+            timePeriodType: 'dr'
+        };
+
+        const filterValue = {
+            filters: {
+                ...defaultFilters,
+                ...categoryFilter,
+                ...locationFilter,
+                ...timePeriodFilter,
+                ...awardTypeFilter
+            },
+            version: REQUEST_VERSION
+        };
+
+        let tempHash = generateUrlHash(filterValue);
+        tempHash.promise
+            .then((results) => {
+                const hashData = results.data;
+                window.open(`/search?hash=${hashData.hash}`, '_blank');
+            })
+            .catch((error) => {
+                console.log(error);
+                if (isCancel(error)) {
+                    // Got cancelled
+                }
+                else if (error.response) {
+                    // Errored out but got response, toggle noAward flag
+                    tempHash = null;
+                }
+                else {
+                    // Request failed
+                    tempHash = null;
+                    console.log(error);
```

Comment:
should we log this same error twice in this case? or just keep the first console.log(error)?